Sources/Widgets/Core/AbstractWidgetFactory/index.js

```diff
@@ -153,13 +153,13 @@ function vtkAbstractWidgetFactory(publicAPI, model) {
   // Event Widget API
   // --------------------------------------------------------------------------
   let unsubscribe = NoOp;
-  publicAPI.delete = macro.chain(publicAPI.delete, unsubscribe);
+  publicAPI.delete = macro.chain(publicAPI.delete, () => unsubscribe());
```

Comment:
Do you really need to wrap it into another function?

Reply:
Oh, I see... Sorry, you are right!

Reply:
Yes, because macro.chain will get the reference to `NoOp` here rather than the correct unsubscribe function.

Reply:
Haha github didn't show me your second message until after I posted.